ui/tests/acceptance/settings/mount-secret-backend-test.js

```diff
@@ -301,26 +305,110 @@ module('Acceptance | settings/mount-secret-backend', function (hooks) {
+        .dom('[data-test-search-select-with-modal]')
+        .doesNotExist('for type ssh, the modal field does not render.');
+      // cleanup
+      await runCmd(`delete identity/oidc/key/some-key`);
+    });
+
+    test('it allows a user with permissions to oidc/key to create an identity_token_key', async function (assert) {
+      for (const engine of WIF_SECRET_ENGINES) {
+        const path = `secrets-adminPolicy-${engine}`;
+        const secrets_admin_policy = adminPolicy();
+        const secretsAdminToken = await runCmd(
+          tokenWithPolicyCmd(`secrets-admin-${path}`, secrets_admin_policy)
+        );
+
+        await logout.visit();
+        await authPage.login(secretsAdminToken);
+        await page.visit();
+        await page.selectType(engine);
+        await page.path(path);
+        await click(GENERAL.toggleGroup('Method Options'));
+        await clickTrigger('#key');
+        // create new key
+        await fillIn(GENERAL.searchSelect.searchInput, 'new-key');
+        await click(GENERAL.searchSelect.options);
+        assert.dom('#search-select-modal').exists('modal with form opens');
+        assert.dom('[data-test-modal-title]').hasText('Create new key', 'Create key modal renders');
+
+        await click(OIDC.keySaveButton);
+        assert.dom('#search-select-modal').doesNotExist('modal disappears onCancel');
+        assert.dom(GENERAL.searchSelect.selectedOption()).hasText('new-key', 'new-key is now selected');
+        await page.submit();
+        assert
+          .dom(GENERAL.latestFlashContent)
+          .hasText(`Successfully mounted the aws secrets engine at ${path}.`);
+        // ARG TODO should redirect to the mount config page. Later PR to address this.
+        // assert.strictEqual(
+        //   currentURL(),
+        //   `/vault/cluster/secrets/backend/${path}/configuration`,
+        //   'After mounting, redirects to secrets configuration page.'
+        // );
+
+        // cleanup
+        await runCmd(`delete sys/mounts/${path}`);
+        await runCmd(`delete identity/oidc/key/some-key`);
+        await runCmd(`delete identity/oidc/key/new-key`);
+      }
+    });
+
+    test('it allows user with NO access to oidc/key to manually input an identity_token_key', async function (assert) {
+      for (const engine of WIF_SECRET_ENGINES) {
+        const path = `secrets-noOidcAdmin-${engine}`;
+        const secrets_noOidcAdmin_policy = noOidcAdminPolicy();
+        const secretsNoOidcAdminToken = await runCmd(
+          tokenWithPolicyCmd(`secrets-noOidcAdmin-${path}`, secrets_noOidcAdmin_policy)
+        );
+
+        await logout.visit();
+        await authPage.login(secretsNoOidcAdminToken);
+        await page.visit();
+        await page.selectType(engine);
+        await page.path(path);
+        await click(GENERAL.toggleGroup('Method Options'));
+        // type-in fallback component to create new key
+        await fillIn('[data-test-input-search="key"]', 'manual-key');
```

Comment:
Could you also add a test for checking that the fallback component has the correct place holder text? That probably makes more sense to live in that component's integration test instead of there 🤔